core-bundle/tests/Security/Authentication/FrontendPreviewAuthenticatorTest.php

```diff
@@ -306,9 +306,9 @@ public function testAuthenticatesAFrontendUserWithUnpublishedElementsHidden(): v
         $authenticator = $this->getAuthenticator($session, $tokenStorage, $userProvider);
 
         $this->assertTrue($authenticator->authenticateFrontendUser('foobar', false));
-        $this->assertTrue($session->has(FrontendUser::SECURITY_SESSION_KEY));
+        $this->assertTrue($session->has('_security_contao_frontend'));
```

Comment:
If `FrontendPreviewAuthenticator::SECURITY_SESSION_KEY` were not private, we could use it here. But I assume you have made it private so we can remove it later?

Reply:
Correct.

Reply:
But what do we need the constant for then? We might as well use the string in the `FrontendPreviewAuthenticator` class.

Reply:
Yeah we could… fine with me 🙃

Reply:
Changed in 737b13c98855376ed51ffc4361fa8476c8953edb then.